mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -343,12 +343,9 @@ function resultsToTables(results, target, uri) {
                     }
                     newalbum = [];
                     newtlids = [];
-                    if (results[i].album) {
-                        coversList.push([results[i].album, i]);
-                    }
                 } //newalbum length
                 if (results[i].album) {
-                    coversList.push([results[i].album, i]);
+                    coversList.push([results[i].uri, i]);
```

Comment:
Did you mean to move this outside the `if`? In the case of there being only one track in the album, the cover art is not displayed. Or is it requested anyway to be used in another view? I admit I'm not 100% familiar with this code.